### ISSUE TITLE
Startup directory doesn't work

### ISSUE THREAD

**Issue by thany** [cmderdev/cmder#1303]
When I've got a Cmder window open, and right-click in the tab bar to select *New console*, the startup directory chosen in the dialog that appears isn't respected. It **always** starts in my home directory (`C:\Users\username`) no matter what options I choose. It also doesn't matter if I do this from an elevated Cmder or not.

I've updated literally minutes ago, so everything should be up-to-date.
Cmder version 1.3.2
ConEmu version 161206
Windows 10 x64

**Comment by kibbled** [comment]
Same here on Windows 7. Also if I close the second tab it crashes Cmder but not if I close the first tab after opening a second. Running latest build 161206 64-bit stable

**Comment by ZenToad** [comment]
Same problem here.  Windows 10

**Comment by Saturate** [comment]
Some issue for me

**Comment by thany** [comment]
Looks like the issue is in the software then, not in the user 🙂

**Comment by angly-cat** [comment]
TL;DR: delete `:d:%USERPROFILE%` from cmder task's command.

Actually, it works as intended. And by that I mean that new Cmder actually starts within specified "Startup directory for new process" and **then** it's working directory gets changed to whatever else.

So, in a task's command that starts cmder you can see something like `-new_console:d:"%USERPROFILE%"`. According to [`-new_console` syntax](https://conemu.github.io/en/NewConsole.html#syntax), `d` flag is for changing working directory. So, for that command to not change directory just remove that switch and this issue will be fixed for you.

**Comment by thany** [comment]
I haven't added that to the command, so whatever the bug exactly is, is caused by default behaviour. Ergo, it still needs a fix and the suggestion you provide is merely a workaround.

And with al ldue respect, "it works as intended" in my mind reads as "it is intended wrongly" ;)

**Comment by thany** [comment]
Can I ask why this issue is closed? Has it been resolved in any way?

**Comment by angly-cat** [comment]
It has been in #1504, as you can see right above your message.

**Comment by drunkwretch** [comment]
@angly-cat  thx  save me
